src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div>
+      <div className="group-one">
```

Comment:
- [ ] **[OPTIONAL]** Instead of having to repeat our self for each button, since all buttons share the same behavior, we can create a 2D array for grouping the buttons and then map through this array to dynamically create the `Button` components.

Reply:
I will consider this in my next milestone. Thank you for the hint.